src/api/lib/tasks/databases.rake

```diff
@@ -83,6 +83,20 @@ namespace :db do
       end
       File.open("#{Rails.root}/db/structure.sql", "w+") { |f| f << new_structure }
     end
+
+    desc "Verify that structure.sql in git is up to date"
+    task verify: :environment do
+      puts "Running rails db:migrate"
+      Rake::Task["db:migrate"].invoke
+      puts "Diffing the db/structure.sql"
+      sh %{git diff --quiet db/structure.sql} do |ok, _|
+        unless ok
+          abort "Generated structure.sql differs from structure.sql stored in git. " +
+            "Please eun rake db:migrate and check the differences."
+        end
+      end
+      puts "Everything looks fine!"
```

Comment:
Actually I prefer how it is right now.

In stdout one would see
```
"Diffing the db/structure.sql"
"Everything looks fine!" 
```

That should be good enough. Don't you think?